package.json

```diff
@@ -33,7 +33,9 @@
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
     "test:integration:ci": "jest --colors ./src/integration/pages/**/*.test.js",
```

Comment:
We could include this in the test:integration command since it is super fast and already hooked into local npm test and in CI